build_tools/customizations.rb

```diff
@@ -114,16 +114,8 @@ def dynamodb_example_deep_transform(subsegment, keys)
       end
     end
 
-    %w(Lambda LambdaPreview).each do |svc_name|
-      api(svc_name) do |api|
-        api['metadata']['serviceId'] ||= 'Lambda Preview' if svc_name == 'LambdaPreview'
-        api['shapes']['Timestamp']['type'] = 'timestamp'
-      end
-
-      doc('lambda') do |docs|
-        docs['shapes']['Blob']['refs']['UpdateFunctionCodeRequest$ZipFile'] =
-          "<p>.zip file containing your packaged source code.</p>"
-      end
+    api('Lambda') do |api|
```

Comment:
Just to confirm, when you run `rake build:aws-sdk-lambda`, there is no diff correct?

Reply:
Yup, that is correct. No diff generated when I ran the rake build.